CollAction/Views/Manage/Index.cshtml

```diff
@@ -9,95 +9,152 @@
     ViewData["Title"] = "Manage your account";
 }
 
+<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
I think jquery and bootstrap styling are already present in the current codebase. Can you check and see what's already there to use?

Reply:
Additionally, if you load from different sources, content security policy will start blocking things.

Reply:
Yes I know sorry this is WIP, was supposed to remove it ;)